numpy/doc/indexing.py

```diff
@@ -1,4 +1,5 @@
-"""==============
+"""
+==============
 Array indexing
 ==============
 
```

Comment:
Note to reviewers: this should be checked in the help, html and ipython rendering

Reply:
The other files in this directory are all wrapped this way too, so I think fine to skip the check.